modules/profiles/fhs.nix

```diff
@@ -16,8 +16,13 @@
         fi
       done
     }
-    sw=/nix/var/nix/gcroots/current-system/sw
```

Comment:
And also fix ldso as part of the changelog?

Reply:
The `CHANGELOG.md` file contains changes to the `trilby-cli` utility itself, the Trilby changelog should ideally be reflected in the commit messages.

I've updated the unhelpful "fix FHS" message to be more clear about what it does.